src/App.css

```diff
@@ -3,15 +3,11 @@
 }
 
 @property --angle {
-  syntax: "<angle>";
+  syntax: '<angle>';
   initial-value: 0deg;
   inherits: false;
 }
 
-.MuiSvgIcon-root {
-  color: #DC3CAA;
-}
```

Comment:
I figured as we are making new changes and bringing in mui icons, we would rip the bandaid off and if you see an icon that is suppose to be this color, we can change it on the icon itself and not globally like this was doing. 
CC: @cullenpu @dmkirshon @scandycuz